src/components/ViewDialog/ViewDialog.tsx

```diff
@@ -46,7 +43,7 @@ export const ViewDialog = ({ row }: ViewDialogProps) => {
       <DialogTrigger className="w-full">
         <DropdownMenuItem onSelect={(e) => e.preventDefault()}>View</DropdownMenuItem>
       </DialogTrigger>
-      <DialogContent className="max-w-3xl max-h-5/6 overflow-auto">
+      <DialogContent className="max-w-3xl max-h-[85vh] overflow-auto">
```

Comment:
Is this change is related with changing `any` types to specific types? 🤔

Reply:
Found this issue while checking types here. This change does not relate to `any` issue

Reply:
what is this change? I don't understand this change, I would suggest to extract it into separated commit an PR

Reply:
It changed the max height of DialogContent in ViewDialog component to 85vh. Will remove this